index.html

```diff
@@ -225,6 +225,57 @@ <h4>Install the client on your headset</h4>
                             </div>
                         </div>
 
+                        <div class="row gx-0 my-5 justify-content-center">
+                            <h4>Settings guide</h4>
```

Comment:
Settings guide should not be in the getting started section, and could be a separate one.

Getting started should not involve any manual configuration and defaults should be good enough.